src/privatesend/privatesend-client.cpp

```diff
@@ -1434,21 +1435,28 @@ bool CPrivateSendClientSession::MakeCollateralAmounts(const CompactTallyItem& ta
         return false;
     }
 
-    CTransactionBuilder txBuilder(mixingWallet, tallyItem);
+    CWallet* pwallet = GetWallet(mixingWallet.GetName());
+
+    if (!pwallet) {
+        LogPrint(BCLog::PRIVATESEND, "CPrivateSendClientSession::%s -- Couldn't get wallet pointer\n", __func__);
+        return false;
+    }
+
+    CTransactionBuilder txBuilder(pwallet, tallyItem);
 
     LogPrint(BCLog::PRIVATESEND, "CPrivateSendClientSession::%s -- Start %s\n", __func__, txBuilder.ToString());
 
     // Skip way too tiny amounts. Smallest we want is minimum collateral amount in a one output tx
-    if (!txBuilder.CouldAddOutput(CPrivateSend::GetCollateralAmount())) {
+    if (!txBuilder.CouldAddOutput(CPrivateSend::GetMinCollateralAmount())) {
```

Comment:
Why is GetCollateralAmount getting refactored in this commit?

Reply:
Hmm good question, guess i just thought it makes sense to give it better name in the same run as we introduce it in `interfaces`. Concerns?

Reply:
Yeah, should minimize the diff on that imo. Please revert. I'm not against it in another PR though